app/models/org.rb

```diff
@@ -171,9 +178,15 @@ def self.default_orgs
               count(users.id) as user_count")
   }
 
-  before_validation :set_default_feedback_email_subject
+  # =============
+  # = Callbacks =
+  # =============
+
+  # this is basically checking the file-store for the dragonfly image each time
+  # before we validate in-case the db/system have moved and the link to the logo files
+  # is therefore different
+  # TODO: evaluate the need for this
   before_validation :check_for_missing_logo_file
```

Comment:
This one is a wierd-case.

The method effectively checks the validity of the image-file each time we save an org, and then does a seccondary search for the file if it cannot find the logo, finally nullifying the UID if it cannot find one.:
```
  # If the physical logo file is no longer on disk we do not want it to prevent the
  # model from saving. This typically happens when you copy the database to another
  # environment. The orgs.logo_uid stores the path to the physical logo file that is
  # stored in the Dragonfly data store (default is: public/system/dragonfly/[env]/)
  def check_for_missing_logo_file
    if self.logo_uid.present?
      data_store_path = Dragonfly.app.datastore.root_path

      if !File.exist?("#{data_store_path}#{self.logo_uid}")
        # Attempt to locate the file by name. If it exists update the uid
        logo = Dir.glob("#{data_store_path}/**/*#{self.logo_name}")
        if !logo.empty?
          self.logo_uid = logo.first.gsub(data_store_path, "")
        else
          # Otherwise the logo is missing so clear it to prevent save failures
          self.logo = nil
        end
      end
    end
  end
```

I don't think there's a way to do the same thing with either an accessor or default, so if we want to retain this logic, we'll need to keep the callback here.  We may find that this is unneccissary if we add some abstraction on ActiveStorage which would remove the reference to the file if it cannot find it.

Reply:
We can ditch this code. If I remember correctly it was an attempt to try and find the logo (back when logos were stored on local disk) when copying the DB from one machine to another